pkg/cmdutil/userio/interop.go

```diff
@@ -0,0 +1,43 @@
+package userio
+
+import (
+	"fmt"
+
+	tea "github.com/charmbracelet/bubbletea"
+	"github.com/charmbracelet/lipgloss"
+)
+
+type InputCompleted struct {
+	model tea.Model
+}
+
+type nonInteractiveHandler struct {
+	streams *IOStreams
+}
+
+func InfoLog(message string) string {
+	style := lipgloss.NewStyle().Foreground(lipgloss.Color("123"))
+	return fmt.Sprintf("%s %s", style.Render("INFO:"), message)
+}
+func WarnLog(message string) string {
+	style := lipgloss.NewStyle().Foreground(lipgloss.Color("227"))
```

Comment:
You better create style using `nih.streams.out.NewStyle()`, it will adapt to the terminal colorscheme automatically (to my knowledge). Also, I would propose to create a style scheme struct similar to: pkg/cmdutil/userio/styles.go:24

Reply:
Is this required? The official components just use the normal lipgloss styles https://github.com/charmbracelet/bubbles/blob/master/list/style.go

Reply:
I'm not 100% sure, it might be important only for specific cases: https://github.com/charmbracelet/lipgloss#custom-renderers.
I think it make sense to still refactor it for consistency at least, if it's not too time consuming

Reply:
I've moved them into the styles.go, I'll leave out the custom rendering for now if it doesn't seem to need it for our use-case